packages/scripts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Master
 
+### Breaking Changes
```

Comment:
I think, we are forced to do a major version bump in case someone was using the old file format with the list of script dependencies. New stuff can be added later as new features using minor version upgrades.